war/src/main/resources/templates/scores/details.html

```diff
@@ -87,10 +92,26 @@ <h3>Probes results</h3>
             <tr data-th-each="item: ${plugin.details}" data-th-with="probeResult = ${item.value}">
                 <td><code data-th-text="${probeResult.id}" class="label"></code></td>
                 <td>
-                    <ion-icon data-th-if="${probeResult.status.name} == 'SUCCESS'" class="success" name="success"></ion-icon>
-                    <ion-icon data-th-unless="${probeResult.status.name} == 'SUCCESS'" class="failure" name="failure"></ion-icon>
+                    <ion-icon data-th-if="${probeResult.status.name} == 'SUCCESS'" class="success"
+                              name="success"></ion-icon>
+                    <ion-icon data-th-unless="${probeResult.status.name} == 'SUCCESS'" class="failure"
+                              name="failure"></ion-icon>
+                </td>
+                <td>
+                    <span
+                        th:if="${probeResult.id == 'security' && probeResult.message != 'No known security vulnerabilities.'}">
+                        <span th:each="securityNumber : ${#strings.arraySplit(probeResult.message, ' ,')}"
+                              th:inline="text">
+                            <a th:text="${securityNumber}"
+                               th:href="'https://www.jenkins.io/security/issue/' + ${securityNumber}"  >
+                            </a>
```

Comment:
This looks ok but some values are not the security issue id but the link to the advisory.
I'm looking for a plugin with such a result.